@joecorall I see https://github.com/Islandora-Devops/isle-buildkit/pull/289 has been merged.

I tried making the changes in this PR then 
`make starter TAG=2.0.1 CODEBASE_PACKAGE=islandora/islandora-starter-site:1.1.0`

The make process got a fair way but then errored:
```
 [success] Installation complete.
docker compose exec -T drupal with-contenv bash -lc "drush -l https://islandora.traefik.me user:role:add fedoraadmin admin"
In EntityTypeManager.php line 139:                     
  The "features_bundle" entity type does not exist.  
make[1]: *** [starter-finalize] Error 1
make: *** [starter] Error 2
```

`docker compose exec -T drupal with-contenv bash -lc "drush -l https://islandora.traefik.me ws"`
gives
```
In EntityTypeManager.php line 139:
  The "features_bundle" entity type does not exist.  
```
as does `drush cr` etc.

--- a/sample.env
+++ b/sample.env
@@ -94,7 +94,7 @@ REPOSITORY=islandora
 
 # The version of the isle-buildkit images, non isle-buildkit images have
 # their versions specified explicitly in their respective docker-compose files.
-TAG=2.0.1
+TAG=2.0.2
 
 ###############################################################################
 # Exposed Containers & Ports

--- a/sample.env
+++ b/sample.env
@@ -94,7 +94,7 @@ REPOSITORY=islandora
 
 # The version of the isle-buildkit images, non isle-buildkit images have
 # their versions specified explicitly in their respective docker-compose files.
-TAG=2.0.2
+TAG=2.0.3
 
 ###############################################################################
 # Exposed Containers & Ports

